src/pe_reports/metrics.py

```diff
@@ -12,25 +22,571 @@ def __init__(self, start_date, end_date, org_uid):
+            .count()
+            .nlargest(10)
+            .reset_index(name="count")
+        )
+        return dark_web_content
+
+    def dark_web_count(self):
+        """Get total number of dark web mentions."""
+        return len(self.dark_web_mentions.index)
+
+    def dark_web_date(self):
+        """Get dark web mentions by date."""
+        dark_web_mentions = self.dark_web_mentions
+        dark_web_date = dark_web_mentions[["date"]]
+        dark_web_date = (
+            dark_web_date.groupby(["date"])["date"].count().reset_index(name="Count")
+        )
+        return dark_web_date
+
+    def dark_web_most_act(self):
+        """Get most active posts."""
+        dark_web_mentions = self.dark_web_mentions
+        dark_web_most_act = dark_web_mentions[["comments_count", "title", "content"]]
+        dark_web_most_act = dark_web_most_act[
+            dark_web_most_act["comments_count"] != "NaN"
+        ]
+        dark_web_most_act = dark_web_most_act.rename(
+            columns={"comments_count": "Comments Count"}
+        )
+        dark_web_most_act["Comments Count"] = (
+            dark_web_most_act["Comments Count"].astype(float).astype(int)
+        )
+        dark_web_most_act = dark_web_most_act.sort_values(
+            by="Comments Count", ascending=False
+        )
+        dark_web_most_act = dark_web_most_act[:5]
+        dark_web_most_act["content"] = dark_web_most_act["content"].str[:100]
+        dark_web_most_act = dark_web_most_act.rename(
+            columns={"title": "Title", "content": "Content"}
+        )
+        dark_web_most_act["Title"] = dark_web_most_act["Title"].str[:50]
+        return dark_web_most_act
+
+    def dark_web_sites(self):
+        """Get mentions by dark web sites (top 10)."""
+        dark_web_mentions = self.dark_web_mentions
+        dark_web_sites = dark_web_mentions[["site"]]
+        dark_web_sites = (
+            dark_web_sites.groupby(["site"])["site"]
+            .count()
+            .nlargest(10)
+            .reset_index(name="count")
+        )
+        dark_web_sites = dark_web_sites.rename(
+            columns={"site": "Site", "count": "Count"}
+        )
+        return dark_web_sites
+
+    def dark_web_tags(self):
+        """Get dark web notable tags."""
+        dark_web_mentions = self.dark_web_mentions
+        dark_web_tags = dark_web_mentions[["tags"]]
+        dark_web_tags = dark_web_tags[dark_web_tags["tags"] != "NaN"]
+        dark_web_tags = (
+            dark_web_tags.groupby(["tags"])["tags"]
+            .count()
+            .nlargest(8)
+            .reset_index(name="Events")
+        )
+        dark_web_tags["tags"] = dark_web_tags["tags"].str.strip("{}")
+        dark_web_tags["tags"] = dark_web_tags["tags"].str.replace(
+            ",", ", ", regex=False
+        )
+        dark_web_tags = dark_web_tags.rename(columns={"tags": "Tags"})
+        return dark_web_tags
+
+    def top_cve_table(self):
+        """Get top cves."""
```

Comment:
Use proper capitalization for acronyms in docstrings:
```suggestion
        """Get top CVEs."""
```